templates/page.html

```diff
@@ -179,25 +179,23 @@ <h3>Select a REANA cluster...</h3>
             <div class="line"><span class="prompt">$</span>export REANA_ACCESS_TOKEN=XXXXXXX</div>
           </div>
         </figure>
-        <h3>...or install your own</h3>
+        <h3>...or install your own locally</h3>
```

Comment:
This change is in prototype state, there are display issues that I didn't yet fix because in my eyes the content is not clear. I have the following doubts:
1. Should this be using `reana-dev`? If so:
   a. is it ok installing from git?
2. Should we just show pure `helm` installation? If so:
   a. Do we assume there is a Kubernetes cluster up and running?
   b. If we want to create one with `kind` there are many subtilities which are hidden behind `reana-dev` which would make it complicated
3. Do we do a mix, `reana-dev` to create the cluster, and use pure `helm` for the installation?

Reply:
> Should we just show pure helm installation?

Yes, I think the option 2, showing pure helm installation, is appropriate for the target audience in this box. It could go like:

```
$ # create Kubernetes cluster (or use your own!)
$ kind create cluster 
$ # deploy REANA using Helm
$ helm repo ...
$ helm install...
# ...and follow displayed post-installation instructions 
# ...to obtain your token
```

> If we want to create one with kind there are many subtilities which are hidden behind reana-dev which would make it complicated

P.S. Regarding cluster creation, when people only use Helm with our released charts, it should be enough to just create pure Kind one without?

Reply:
> P.S. Regarding cluster creation, when people only use Helm with our released charts, it should be enough to just create pure Kind one without?

That is actually the "problem". We need for sure to at least [open ports](https://github.com/reanahub/reana/blob/9deb09f740a6e18dd26fb2c7c7bab1aee935c02a/reana/reana_dev/cluster.py#L103) and this [can only be done through config file](https://kind.sigs.k8s.io/docs/user/configuration/#getting-started). We could have a sample file with simple config and `wget` it here?

Reply:
`wget github.com/reanahub/reana/.../kind-cluster-localhost.yaml` might do, similarly we could `wget github.com/reanahub/reana/.../scripts/create-admin-user.sh` so that everything would be ready for localhost deployment (think workshop) in 2-3 easy commands...

Reply:
Added in https://github.com/reanahub/reana/pull/410